compiler/src/dmd/parse.d

```diff
@@ -4596,6 +4596,11 @@ class Parser(AST, Lexer = dmd.lexer.Lexer) : Lexer
                  */
                 if (tpl)
                 {
+                    if (storage_class & STC.override_)
+                        error(loc, "a function template cannot be `override`");
```

Comment:
Maybe a bit more info would come in handy. Something along the lines of: "Function templates are never virtual, therefore they cannot override anything." and then add an error supplemental stating: "`override` used to be accepted and ignored on function templates. Please delete the `override` keyword."